### PR TITLE
[BE] Remove `"$PACKAGE_TYPE" == 'conda'` logic from build scripts

### PR DESCRIPTION
**Pull request by atalman**
Please see: https://github.com/pytorch/pytorch/issues/138506

### CODE REVIEW

**Review by huydhn — APPROVED** [review]
LGTM!

File: .circleci/scripts/binary_linux_build.sh [hunk 1]
```diff
@@ -16,9 +16,7 @@ if [[ "${DESIRED_CUDA}" =~ cu1[1-2][0-9] ]]; then
 fi
 
 # Parse the parameters
-if [[ "$PACKAGE_TYPE" == 'conda' ]]; then
-  build_script='conda/build_pytorch.sh'
```
Comment (malfet) [inline .circleci/scripts/binary_linux_build.sh:20]: Why not delete `conda/build_pytorch.sh` script as well?

File: .circleci/scripts/binary_windows_build.sh [hunk 2]
```diff
@@ -20,12 +20,8 @@ echo "Free space on filesystem before build:"
 df -h
 
 pushd "$BUILDER_ROOT"
-if [[ "$PACKAGE_TYPE" == 'conda' ]]; then
-    ./windows/internal/build_conda.bat
```
Comment (malfet) [inline .circleci/scripts/binary_windows_build.sh:24]: Also, delete `windows/internal/build_conda.bat`

**Review by malfet — APPROVED** [review]
LGTM, but please delete dangling scripts (windows/internal/build_conda.bat` and `conda/build/build_pytorch.sh`

File: .circleci/scripts/binary_linux_build.sh (hunk 1 above)
Comment (atalman) [reply .circleci/scripts/binary_linux_build.sh:20]: I believe its already deleted by : https://github.com/pytorch/builder/pull/2045

File: .circleci/scripts/binary_windows_build.sh (hunk 2 above)
Comment (atalman) [reply .circleci/scripts/binary_windows_build.sh:24]: Yes posted: https://github.com/pytorch/builder/pull/2047

**Pull request closed without merging**